client/src/components/lower-info-area/lower-container.vue

```diff
@@ -76,37 +82,39 @@
       </v-col>
     </v-row>
 
-    <v-row no-gutters class="contact-section">
-      <v-col cols="6">
-        <v-container class="pa-9">
-          <v-row>
-            <v-col cols="12" class="h3 info-underline px-0">Need more information?</v-col>
-            <v-col cols="12" class="px-0">
-              <a :href="needMoreInfo" target="_blank" class="white--text">
-                Learn more about requesting a name
-              </a>
-            </v-col>
-          </v-row>
-        </v-container>
-      </v-col>
-      <v-col cols="6">
-        <v-container class="pa-9">
-          <v-row alinn="start" align-content="start" align-items="start">
-            <v-col cols="11" class="h3 info-underline px-0">Contact Us</v-col>
-            <v-col cols="12" class="px-0">
-              For support or questions about this application contact us at:
-            </v-col>
-            <ul class="contact-info-list">
-              <li>Toll Free: <a href="tel:1-877-370-1033" class="white--text">1-877-370-1033</a></li>
-              <li>Phone: <a href="tel:250-370-1033" class="white--text">250-370-1033</a></li>
-              <li>Email: <a href="mailto:bcregistries@gov.bc.ca" class="white--text">bcregistries@gov.bc.ca</a></li>
-            </ul>
-            <v-col cols="12" class="pb-0 px-0 fw-700">Hours of Operation:</v-col>
-            <v-col cols="12" class="pt-0 px-0">Monday to Friday, 8:30am - 4:30pm Pacific Time</v-col>
-          </v-row>
-        </v-container>
-      </v-col>
-    </v-row>
+    <div class="contact-section">
+      <v-row no-gutters class="content-container">
+        <v-col cols="6">
+          <v-container class="pa-9">
+            <v-row>
+              <v-col cols="12" class="h3 info-underline px-0">Need more information?</v-col>
+              <v-col cols="12" class="px-0">
+                <a :href="needMoreInfo" target="_blank" class="white--text">
+                  Learn more about requesting a name
+                </a>
+              </v-col>
+            </v-row>
+          </v-container>
+        </v-col>
+        <v-col cols="6">
+          <v-container class="pa-9">
+            <v-row alinn="start" align-content="start" align-items="start">
+              <v-col cols="12" class="h3 info-underline px-0">Contact Us</v-col>
+              <v-col cols="12" class="px-0">
+                For support or questions about this application contact us at:
+              </v-col>
+              <ul class="contact-info-list">
```

Comment:
NB to all devs: If we come across another use of this contact info list, let's pull it out into a separate component and reuse this code.